support-frontend/assets/components/sepaForm/SepaForm.tsx

```diff
@@ -44,27 +46,29 @@ export function SepaForm({
 	updateAddressCountry,
 	updateIban,
 	updateAccountHolderName,
+	errors,
 }: SepaFormProps): JSX.Element {
 	return (
 		<div css={containerStyles}>
 			<h3 css={headerStyles}>Your account details</h3>
 			<Stack cssOverrides={fieldsContainerStyles} space={3}>
 				<div>
 					<TextInput
-						id="sepa-account-holder-name-input"
+						id="accountHolderName"
 						data-testid="sepa-account-holder-name-input"
 						data-qm-masking="blocklist"
 						optional={false}
 						hideLabel={false}
 						label="Bank account holder name"
 						maxLength={40}
 						value={accountHolderName ?? undefined}
+						error={errors.accountHolderName?.[0]}
 						onChange={(e) => updateAccountHolderName(e.target.value)}
 					/>
 				</div>
 				<div>
 					<TextInput
-						id="sepa-account-number"
```

Comment:
Should we pull out the data here and map an object instead to prevent the repetition?